applications_tests/dem_3d/CMakeLists.txt

```diff
@@ -14,11 +14,19 @@ string(TOLOWER ${CMAKE_BUILD_TYPE} CMAKE_LOWER_BUILD_TYPE)
 DEAL_II_PICKUP_TESTS()
 
 if (CMAKE_BUILD_TYPE STREQUAL "Debug" )
+       	set_tests_properties(dem_3d/packing_in_cylinder.debug PROPERTIES TIMEOUT 3000) 
+       	set_tests_properties(dem_3d/packing_in_ball_dynamic_contact.debug PROPERTIES TIMEOUT 3000)
+       	set_tests_properties(dem_3d/sliding_in_box.debug PROPERTIES TIMEOUT 3000)
+    if (${DEAL_II_VERSION_MAJOR} LESS 10)
        	set_tests_properties(dem_3d/packing_in_box.mpirun=1.debug PROPERTIES TIMEOUT 3000) 
        	set_tests_properties(dem_3d/packing_in_box.mpirun=2.debug PROPERTIES TIMEOUT 3000) 
        	set_tests_properties(dem_3d/packing_in_ball.mpirun=1.debug PROPERTIES TIMEOUT 3000) 
        	set_tests_properties(dem_3d/packing_in_ball.mpirun=2.debug PROPERTIES TIMEOUT 3000) 
-       	set_tests_properties(dem_3d/packing_in_cylinder.debug PROPERTIES TIMEOUT 3000) 
-       	set_tests_properties(dem_3d/packing_in_ball_dynamic_contact.debug PROPERTIES TIMEOUT 3000)
-       	set_tests_properties(dem_3d/sliding_in_box.debug PROPERTIES TIMEOUT 3000)
+
+    else()
+	    set_tests_properties(dem_3d/packing_in_box.mpirun=1.threads=0.debug PROPERTIES TIMEOUT 3000)
+      	set_tests_properties(dem_3d/packing_in_box.mpirun=2.threads=0.debug PROPERTIES TIMEOUT 3000) 
+       	set_tests_properties(dem_3d/packing_in_ball.mpirun=1.threads=0.debug PROPERTIES TIMEOUT 3000) 
+       	set_tests_properties(dem_3d/packing_in_ball.mpirun=2.threads=0.debug PROPERTIES TIMEOUT 3000) 
```

Comment:
@blaisb Yes, I have changed the target name.
It might not have been the best decision to unconditionally add the `.threads=0` (it simply saves another set of if clauses to decide whether `.mpirun=N` or `.threads=N` has to be present). So i this change is inconvenient we can certainly try to address it in upstream deal.II.

Reply:
@tamiko  I am cool with both solutions (either way). I will adapt myself to the library. Just tell me which way you chose and I'll live with it :).